modules/ROOT/pages/external-organization-access.adoc

```diff
@@ -0,0 +1,39 @@
+= External Organization Access
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The External Access feature allows separate organizations to collaborate in the Anypoint Platform. For example, organizations can share API specifications, connectors, and other assets in the Anypoint Exchange with each other, without making them public.
+
+Administrators control which organizations their users can collaborate with by maintaining a list of trusted organizations in Access Management. Adding an entry to this list means that you trust the other organization and its users, and you acknowledge that you are enabling your users to grant permission to and be granted permissions by the other organization.
+
+Trust must be established both ways before permissions can be granted. For the users of Organization A to share with the users of Organization B, Organization A must list Organization B as a trusted organization, and Organization B must also list Organization A. If either Organization A or Organization B removes the other, all shared permissions are revoked.
+
+Permissions shared between organizations are granted at the organization level. For example, if Organization A grants Organization B permission to view an asset, it grants that permission to every user in Organization B. Organization A does not receive any information about Organization B's users, sub-organizations, or internal layout.
+
+== Add a New Trusted Organization
+
+When you add a new organization to your external access configuration, you establish one half of a trust relationship. After the other organization adds you to its configuration, both organizations are able to share permissions with each other.
```

Comment:
```suggestion
When you add a new organization to your external access configuration, you establish one half of a trust relationship. After the other organization adds yours to its configuration, both organizations can share permissions with each other.
```
"permissions" or "access"? I often see us interchange these, along with other terms, and I always ask myself which is the most precise to this context, and try to use the best one consistently throughout a topic set.

Reply:
Unfortunately, it's both in this context. They get access to assets elsewhere (dependent on what product is using this framework) but they also get user permissions associated with those assets. It is complicated.